README.md

```diff
@@ -1,11 +1,7 @@
-# waves-examples
```

Comment:
local folder can be renamed to generate_audio_local, local is not that informative? local what?  @smallest-admin thoughts?